messaging/src/main/java/io/lantern/messaging/CryptoWorker.kt

```diff
@@ -665,7 +709,7 @@ internal class CryptoWorker(
             // a given combination of from/to (i.e. each the conversation only has one introduction
             // message per distinct contact)
             tx.introductionMessage(senderId, introduction.to.id)?.let {
-                tx.delete(it.path)
+                messaging.deleteLocally(it.detailPath)
```

Comment:
Drive-by, we need to completely delete the old introduction so that it doesn't show up in any indexes and such.